src/components/common/ErrorMsgBox/styles.js

```diff
@@ -4,6 +4,7 @@ export const useStyles = theme => ({
     borderStyle: "solid",
     borderColor: theme.backgroundColor.red,
     padding: "13px 20px",
+    marginTop: "20px !important",
```

Comment:
Do we need `!important`

Reply:
[removed](https://github.com/singnet/refactored-dapp/pull/17/commits/e0979060df071f3f0d74d4a082bf1c3bd9df0e17)